fastlane/lanes/release.rb

```diff
@@ -354,19 +356,28 @@
   # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
   #
   desc 'Performs the final checks and triggers a release build for the hotfix in the current branch'
-  lane :finalize_hotfix_release do |options|
-    ensure_git_branch_is_release_branch
-
-    # Verify that there's nothing in progress in the working copy
+  lane :finalize_hotfix_release do |skip_confirm: false|
+    ensure_git_branch_is_release_branch!
     ensure_git_status_clean
 
-    # Pull the latest hotfix release branch changes
-    git_pull
+    hotfix_version = release_version_current
 
-    UI.important("Triggering hotfix build for version: #{release_version_current}")
-    UI.user_error!('Aborted by user request') unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
+    UI.important("Triggering hotfix build for version: #{hotfix_version}")
+    UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.") unless skip_confirm || UI.confirm('Do you want to continue?')
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/3ea5ce7fb46853fc78e2b0ca2196de39e75cd487/fastlane/lanes/release.rb#L366"></span>Style/IfUnlessModifier: Modifier form of <code>unless</code> makes the line too long.</td>
    </tr>
  </tbody>
</table>